sites/main-site/src/content/blog/2025/paper_v2.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Empowering bioinformatics communities with Nextflow and nf-core"
+subtitle: "Genome Biology review"
+pubDate: 2025-08-05T12:00:00+01:00
+headerImage: "/assets/images/blog/paper_v2/paper_v2.png"
+headerImageAlt: "nf-core logo"
+headerImageDim: [250, 250]
+embedHeaderImage: false
+authors:
+    - "maxulysse"
+    - "jfy133"
+    - "mashehu"
+label:
+    - "maintainers"
+---
+
+## Empowering bioinformatics communities with Nextflow and nf-core
+
+Last year, we [announced](https://bsky.app/profile/nf-co.re/post/3ksi4mr6uyl25) the preprint, but we are now thrilled to share that the latest nf-core community paper in **Genome Biology** is live!
+
+[Empowering bioinformatics communities with Nextflow and nf-core](https://rdcu.be/eyqzX)
```

Comment:
```suggestion
[_Empowering bioinformatics communities with Nextflow and nf-core_](https://doi.org/10.1186/s13059-025-03673-9)
```

Let's not use the readcube thing, that's just for shady tracking shit by the publisher... it's an open access paper so we shouldn't have any issues with people accessing them